src/components/ActionBar/src/ActionBarLayer.vue

```diff
@@ -83,7 +83,7 @@ export default {
 	display: flex;
 	justify-content: space-between;
 	box-sizing: border-box;
-	padding: 24px 24px 58px 24px;
+	padding: 24px 24px 64px 24px;
```

Comment:
You also have to increase ActionBarLayer's padding-bottom above from 146px to 152px.